src/microbe_stage/PlayerMicrobeInput.cs

```diff
@@ -191,6 +191,10 @@ public void ToggleUnbinding()
         {
             stage.HUD.HintText = TranslationServer.Translate("UNBIND_HELP_TEXT");
             control.State = MicrobeState.Unbinding;
+
+            ref var callbacks = ref stage.Player.Get<MicrobeEventCallbacks>();
+
+            callbacks.OnUnbindEnabled?.Invoke(stage.Player);
```

Comment:
I guess this is the best place to put this, but could you add a comment on `OnUnbindEnabled` that it currently only works for the player?